code/game/objects/structures/aliens.dm

```diff
@@ -291,6 +291,7 @@
 		child.forceMove(get_turf(src))
 		// TECHNICALLY you could put non-facehuggers in the child var
 		status = BURST
+		update_icon()
```

Comment:
you should update icon even if there is no child

Reply:
Done